helm_deploy/court-register/templates/_envs.tpl

```diff
@@ -10,15 +10,21 @@ env:
   - name: JAVA_OPTS
     value: "{{ .Values.env.JAVA_OPTS }}"
 
-  - name: SPRING_PROFILES_ACTIVE
-    value: "logstash"
+  - name: JWT_PUBLIC_KEY
```

Comment:
you shouldn't need the `jwt_public_key` - should really set `SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI` instead